support/supportedversion/version.go

```diff
@@ -76,3 +80,43 @@ func IsValidReleaseVersion(version, currentVersion, latestVersionSupported, minS
 
 	return nil
 }
+
+type ocpVersion struct {
+	Name        string `json:"name"`
+	PullSpec    string `json:"pullSpec"`
+	DownloadURL string `json:"downloadURL"`
+}
+
+func LookupLatestSupportedRelease(ctx context.Context) (string, error) {
+	prefix := "https://multi.ocp.releases.ci.openshift.org/api/v1/releasestream/4-stable-multi/latest"
```

Comment:
Did we make a decision of defaulting to multiarch? This has an impact particularly for disconnected environments https://issues.redhat.com/browse/MIXEDARCH-292 / https://issues.redhat.com/browse/MIXEDARCH-255

At minimum I thin we should document and communicate this clearly in docs?

Reply:
The cli defaulted to multi, so i brought that over to the webhook. https://github.com/openshift/hypershift/blob/1fc3c32155a1b12fc964812ea310aefd313e07ca/cmd/version/version.go#L22 

If someone is using disconnected, I doubt they'd be using the release image defaulting.